content/blog/build-a-video-thumbnailer-with-pulumi-using-lambdas-containers-and-infrastructure-on-aws/index.md

```diff
@@ -52,31 +46,27 @@ If you're on Windows, run this:
     SET "PATH=%PATH%;%USERPROFILE%.pulumiin"
 
 You'll deploy this app to your own AWS account, so follow the steps
-to [configure your AWS account](https://pulumi.io/install/aws.html).
+to [configure your AWS account]({{< ref "/docs/quickstart/aws" >}}).
```

Comment:
This should probably link to the page on actually getting the AWS account setup, which is now `/docs/reference/clouds/aws/setup.md`

Reply:
Done